.github/workflows/main.yml

```diff
@@ -10,6 +10,7 @@ jobs:
     strategy:
       matrix:
         repo: [ 'tarantool/tarantool']
+        branch: ['master', '2.11', 'release/3.2', 'release/3.3', 'release/3.4']
```

Comment:
I suppose this should be `release/2.11` instead.

Reply:
Fixed